app/src/main/java/org/p2p/wallet/striga/offramp/StrigaOffRampModule.kt

```diff
@@ -0,0 +1,9 @@
+package org.p2p.wallet.striga.offramp
+
+import org.koin.dsl.module
+import org.p2p.core.common.di.InjectionModule
+
+object StrigaOffRampModule : InjectionModule {
+    override fun create() = module {
+    }
```

Comment:
Oops

Reply:
no oops here :)
this is a base to build off-ramp stuff for Eduard and our team